crates/astria-core/src/protocol/genesis/v1alpha1.rs

```diff
@@ -630,15 +750,20 @@ impl FeesError {
             name,
         })
     }
+
+    fn fee_components_conversion(err: FeeComponentError) -> Self {
+        Self(FeesErrorKind::FeeComponentsConversion(err))
+    }
 }
 
 #[derive(Debug, thiserror::Error)]
 #[error("failed ensuring invariants of {}", Fees::full_name())]
 enum FeesErrorKind {
     #[error("field was not set: `{name}`")]
     FieldNotSet { name: &'static str },
+    #[error("conversion of fee components failed")]
+    FeeComponentsConversion(#[source] FeeComponentError),
```

Comment:
The field which failed is contained within the source `FeeComponentError`'s inner `FeeComponentErrorKind`. So, it should still be printed out when printing the source chain. I think it may be better to leave it this way, since this more accurately reflects the level at which the validation failure occurred.